frontend/src/assets/css/variables.scss

```diff
@@ -1,6 +1,7 @@
 :root {
   --white: #ffffff;
   --shadow: 0 2px 4px #0000001a;
+  --sidebar-shadow: 0px 4px 23px 0px #e5e1ff;
```

Comment:
Let's not create a variable for this, this is to specific

Reply:
@Liza-Veis, hey I checked the Figma design and it seems that we are going to use these specific colors(`0px 4px 23px 0px #e5e1ff`) for a lot of shadows in the future - should I save the #e5e1ff as a global color - if so would violet-100 work?

Reply:
Yes